### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by DivyanshSareen**
added typography to style-guide.css

### CODE REVIEW

**Comment by DivyanshSareen** [comment]
messed up some branches, please review the last two commits [7c25a1c](https://github.com/DivyanshSareen/PeaceUI/pull/7/commits/7c25a1c462d27f382b76b9084e995d833aea56b8)

File: components/style-guide.css [hunk 1]
```diff
@@ -219,3 +152,83 @@ body{
...
+}
+
+.sub-title1{
+    font-size: 1rem;
+    font-weight: var(--regular);
+    letter-spacing: 0.15px;
+}
+
+.sub-title2{
+    font-size: 0.875rem;
```
Comment (zoyatanweer) [inline components/style-guide.css:201]: change this into px as it is less than 10.

File: components/style-guide.css [hunk 2]
```diff
@@ -219,3 +152,83 @@ body{
...
+    font-weight: var(--medium);
+    letter-spacing: 0.1px;
+}
+
+.sub-title-grey{
+    font-size: 0.875rem;
```
Comment (zoyatanweer) [inline components/style-guide.css:207]: here too

File: components/style-guide.css [hunk 3]
```diff
@@ -219,3 +152,83 @@ body{
...
+}
+
+.paragraph1{
+    font-size: 1rem;
+    font-weight: var(--regular);
+    letter-spacing: 0.5px;    
+}
+
+.paragraph2{
+    font-size: 0.875rem;
```
Comment (zoyatanweer) [inline components/style-guide.css:220]: change into px.

**Review by zoyatanweer — APPROVED** [review]
do the suggested changes, and you're good to merge. Code looks great.

File: components/style-guide.css (hunk 1 above)
Comment (DivyanshSareen) [reply components/style-guide.css:201]: actually my base font-size is 16 px so this will be equal to 14px

**Review by zoef-yt — APPROVED** [review]
Good code. Good to merge.

**Pull request merged**